content/docs/user-guide/assets/scene-settings/meshes-tab.md

```diff
@@ -0,0 +1,146 @@
+| **Rotation** | Set the orientation offset of the product asset in degrees. This property is most commonly used to reorient a product asset to O3DE's coordinate system. |
+| **Scale** | Set the scale offset of the product asset. O3DE's base unit is one meter. This property is most commonly used to uniformly scale product assets from DCC applications that use centimeters or inches as base units. |
+
+## Level of Detail (LOD)
+
+![The Scene Settings Meshes tab Level of Detail modifier.](/images/user-guide/assets/scene-settings/mesh-lod-modifier.png)
+
+LODs are optimized meshes with progressively lower polygon counts, fewer and smaller textures, and simplified materials. The farther an entity is from the camera, the less detail is required from the meshes that make up the entity. As the entity moves farther from the camera, it swaps to a lower mesh LOD.
+
+You can specify up to five LODs (not including the base mesh) that are numbered \[`0`\] to \[`4`\], with \[`0`\] being the *highest* level of detail. LODs are not required but are recommended because they help get the best performance and visual fidelity across a range of platforms with different hardware capabilities.
+
+* Choose the {{< icon add.svg >}} **Add** button to add an LOD.
+
+* Choose the {{< icon delete.svg >}} **Delete** button to remove an LOD.
+
+* Choose the {{< icon browse-edit-select-files.svg >}} **Selection list** button to specify the meshes to include in the LOD.
+
+{{< note >}}
+You can add `_lod0`, `_lod1`, `_lod2`, `_lod3`, `_lod4`, and `_lod5` as suffixes to your mesh names to automatically add a Level of Detail modifier and assign the meshes to the appropriate LOD slots. `_lod0` is the base mesh with the highest resolution geometry, textures, and materials and is assigned to the base mesh group. `_lod1` is assigned to LOD slot **\[0\]**, `_lod2` is assigned to LOD slot **\[1\]**, and so on.
+{{< /note >}}
+
+## Material
+
+![The Scene Settings Meshes tab Material modifier.](/images/user-guide/assets/scene-settings/material-modifier.png)
+
+The Material modifier helps automatically manage the contents of the `.material` file that corresponds to the mesh group when mesh assets are updated.
+
+A material is a combination of shaders and properties that define the surface of a mesh. Materials contain shader and texture assignments, settings for shader properties such as smoothness, opacity, emissive color, and if necessary, a physics material assignment that defines physical surface properties such as friction.
+
+| Property | Description |
+| - | - |
+| **Update materials** | When active, the texture map file names in the `.material` file are updated to match the texture map names defined in the source asset file. |
+| **Remove unused materials** | When active, materials that are present in the `.material` file that are not defined in the source asset file are removed. |
+
```

Comment:
I have a texture settings doc in progress for this. It will be in the assets section along side this set of topics. I documented the material modifier here, but was having issues getting it to work properly and am wondering if this implementation will be changed soon.